pages/common/ld.md

```diff
@@ -0,0 +1,16 @@
+# ld
+
+> The GNU ELF linker.
+> More information: <https://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_mono/ld.html>.
+
+- Link an object file with no dependencies into an executable:
+
+`ld {{path/to/file.o}} --output {{path/to/output_executable}}`
+
+- Link two object files together:
+
+`ld {{path/to/file1.o}} {{path/to/file2.o}} --output {{path/to/output_executable}}`
+
```

Comment:
Building shared libraries is another common usage, could you insert an example of that here?

Reply:
on it